### ISSUE TITLE
Winner not set in election

### ISSUE THREAD

**Issue by pmk01** [DemocracyClub/yournextrepresentative#1497]
https://candidates.democracyclub.org.uk/elections/local.crawley.tilgate.2016-05-05/

'Mark as elected' does nothing

**Comment by michaeljcollinsuk** [comment]
I can't actually see the "Mark as elected" as I think this was removed after this ticket was created. This was changed so that for FPTP elections, rather than marking a candidates as the winner, we want to get the full result stored - so we added the "Edit results" link to the data timeline. If the results are re-saved, this should mark the correct people as elected based on the votes cast entered.

Having said that, I just went to do this and got a [500 error](https://sentry.io/organizations/democracy-club-gp/issues/2405035070/?project=169287&query=is%3Aunresolved) which is related to when we don't have a `winner_count` value for a ballot. So I've opened a [PR](https://github.com/DemocracyClub/yournextrepresentative/pull/1515) to fix this, and after its been merged and deployed we should be able to save the results again.
